.github/workflows/create-git-tag/action.yml

```diff
@@ -3,20 +3,22 @@ description: Generate a new Git version and push it to the remote repository.
 
 inputs:
   MAINTAIN_USERNAME:
+    description: 'The username of the user that will create the tag.'
```

Comment:
Add a "TODO" comment to remove this file when the old publish is removed

Reply:
Don't know if the relevant because the previous review regard to `Create GitHub release;